### ISSUE TITLE
Can we have a drawing pad or signature pad?

### ISSUE THREAD

**Issue by hmheng** [react-native-community/.github#53]
# Feature Request
There is react-native-signature-signature-pad or some other drawing pad, those are great but they are quite outdated now. May we have them be improved and included in react-native-community package? Also can we have it for RN for Windows and RN for Web support too?

https://github.com/RepairShopr/react-native-signature-capture

**Comment by mikehardy** [comment]
I think the idea of the react-native-community is a collection of people working together on their own software, not a pool of labor that will build some other software on demand

I think the path to having software hosted in react-native-community is more like:

- make sure your software works well with current versions
- commit to maintaining it, so it will see attention from you
- show that to be the case with a reasonable following, and reasonable maintenance for a while
- ask to have your reasonably-well-maintained, working repository hosted under the community umbrella

In short - make useful software and demonstrate a commitment to it.

That's my impression anyway

**Comment by hmheng** [comment]
Hi Mike,

Thanks for your prompt. I didn't expect my suggestion sounds rude to you.
with respect to the community and appreciate what community had done, I didn't treat anyone here as labour. What I suggested is only what I think it might be useful for others, so the entire community can consider to do since the library that I mention is quite dated. Of course, I am still quite new to the RN. 

If there is clear guide how to build plugin for multiple platforms, I would love to do and contribute it with my capacity.

Hope you understand. 

Thank you.

**Comment by mikehardy** [comment]
No worries - I didn't think it was rude, I just wanted to put forward that it's more about help with work vs requests, and I the path I outlined is (I think?) about the path I've seen software take to get there. I'm not aware of clear guides per se, but the path I've taken is to just start working on the software you need. You mentioned a package that had potential but was outdated - start by updating it and go from there I think

**Comment by hmheng** [comment]
Hopefully I have some ideas how to develop it further though. Hope more people can guide me on how to achieve that.

**Comment by kelset** [comment]
👋 @hmheng, to further explain how it works, there is currently this issue -> https://github.com/react-native-community/.github/issues/34

The library you linked is owned by someone other than you, so you should first get this person to give you rights to do that and then we could try and follow what written in the other issue I linked above.

All of this said, what @mikehardy mentioned above is also correct, we are still finding the right balance before writing it in the guidelines - sorry if it caused confusion!
